prime-router/src/main/kotlin/history/azure/DatabaseDeliveryAccess.kt

```diff
@@ -41,8 +42,17 @@ class DatabaseDeliveryAccess(
         organization: String,
         orgService: String?,
     ): Condition {
-        var filter = ACTION.ACTION_NAME.eq(TaskAction.batch)
-            .and(REPORT_FILE.RECEIVING_ORG.eq(organization))
+//      This is a temporary fix to make the daily data page show data for reports routed through the ELR ELIMS topic since reports routed through the ELR-ELIMS topic skip the batch step. There are plans to have a dashboard for receivers that works for all messages routed through RS.
+        val org = BaseEngine.settingsProviderSingleton.findOrganization(organization)
+
+        var filter = if (org?.featureFlags?.contains("ELIMS_DATA") == true) {
+                ACTION.ACTION_NAME.eq(TaskAction.batch)
+                    .or(ACTION.ACTION_NAME.eq(TaskAction.send).and(REPORT_FILE.SCHEMA_TOPIC.eq(Topic.ELR_ELIMS)))
```

Comment:
I would maybe run these in production in metabase?  It doesn't look like report_file.schema_topic is indexed

Reply:
@mkalish Here's a PR showing the explain plan in production before and after we added the receiving org and service index. It doesn't look like it needs an index on the schema_topic.